public/index.html

```diff
@@ -0,0 +1,370 @@
+            <li class="hidden-class nav-li"><a href="#about">About.</a></li>
+            <li class="hidden-class nav-li"><a href="#team">Team.</a></li>
+            <li class="hidden-class nav-li"><a href="#contact">Contact.</a></li>
+        </ul>
+    </nav>
+    <section class="header-section">
+        <header>
+            <img src="./imgs/top-logo.gif" alt="main page logo">
+            <p>Hi there! We're Avengers, <strong>creative design agency</strong>
+                <br>we build <span class="awesome-sm"><strong>awesome</strong></span> mobile and websites</p>
+            <object class="awesome" type="image/svg+xml" data="./imgs/underline.svg"></object>
+        </header>
+    </section>
+    <section class="section-orange">
+        <object class="clip clip-white" type="image/svg+xml" data="./imgs/clipwhite.svg">
+        </object>
+        <a id="services"></a>
+        <h1>SERVICES</h1>
+        <p>Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+        <div class="column-two">
+            <p class="column-left">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+            <p class="column-right">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+        </div>
+        <ul class="flex-container">
+            <li class="flex-item-1">
+                <h2>UI &#38; UX Design</h2>
+                <img src="./imgs/brush.gif" alt="brush logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </li>
+            <li class="flex-item-1">
+                <h2>LABORATORY</h2>
+                <img src="./imgs/erhlenmeyer.gif" alt="flask logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour</p>
+            </li>
+            <li class="flex-item-1">
+                <h2>DEVELOPMENT</h2>
+                <img src="./imgs/gear.gif" alt="gear logo">
+                <p>There are many variation of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </li>
+        </ul>
+        <h2 class="subheader">PRICING TABLES</h2>
+        <section class="table-section flex-container">
+            <table class="flex-item-3">
```

Comment:
The classes of `flex-container` on the containing `section` and `flex-item-3` on the `table`s are misleading as you aren't actually using flex-box to lay the tables out. I see "`flex`" and expect that this is a flex-box layout.